uefi-macros/Cargo.toml

```diff
@@ -15,10 +15,10 @@ rust-version = "1.68"
 proc-macro = true
 
 [dependencies]
-proc-macro2 = "1.0.28"
-quote = "1.0.9"
-syn = { version = "2.0.4", features = ["full"] }
+proc-macro2 = "1.0"
+quote = "1.0"
+syn = { version = "2.0", features = ["full"] }
 
 [dev-dependencies]
-trybuild = "1.0.61"
-uefi = { version = "0.20.0", default-features = false }
+trybuild = "1.0"
+uefi = { version = "0.20", default-features = false }
```

Comment:
I don't think these deps changes are necessary. Specifying a version like "1.2.3" doesn't prevent higher versions like "1.2.4" or "1.3.0" from being selected. It just prevents anything lower from being selected. We could probably relax the `syn` requirement down to `2.0.0`, but the other deps are already at their minimum due to various breaking changes that have been released in deps without bumping the minor version. For example, we have `log = 0.4.5` right now because that's the actual minimum (or was last time I tested it). See https://github.com/rust-osdev/uefi-rs/pull/482

Reply:
Yeah, I thought about it again and agree with you. It was just an idea.. :)